x-pack/plugins/enterprise_search/public/applications/app_search/components/engines/components/meta_engines_table_expanded_row.scss

```
@@ -0,0 +1,22 @@
+
+.meta-engines__source-engines-table {
+  margin: -8px -8px 7px -8px;
```

We should prefer using [EUI size variables](https://github.com/elastic/eui/blob/master/src/global_styling/variables/_size.scss) over static px

```suggestion
  margin: -$euiSizeS -$euiSizeS $euiSizeS -$euiSizeS;
```
```